code/modules/clothing/head/head.dm

```diff
@@ -5,6 +5,7 @@
 	flags_equip_slot = ITEM_SLOT_HEAD
 	w_class = WEIGHT_CLASS_SMALL
 	var/anti_hug = 0
+	var/accuracy_mod = 0.1 //used in gun_system.dm recommended between -1 and 1
```

Comment:
Make zero the default for non-helmet headgear.

Reply:
I must argue against that.

The setup I did was that, if the headgear has no armor, +0.1, if it does have armor +0, and if it has more armor than usual, -0.1

Reply:
Do you see the issue of gaining accuracy by putting on a hat?

Reply:
As the changelog states, people will gain accuracy if they have no hat on. in fact, putting a hat on lowers their accuracy.

The helmet parent has 0. The presumption here is that people are trading protection for accuracy. (and in this case trading style for accuracy in the case of hats.)

Reply:
I added a living accuracy variable.
Give humans a positive value and remove the base helmet bonus.
The base `0.2` accuracy when shooting is removed on my implementation.